bootstrap/frontend/src/test/scala/io/udash/bootstrap/datepicker/UdashDatePickerTest.scala

```diff
@@ -43,23 +43,23 @@ class UdashDatePickerTest extends AsyncUdashFrontendTest {
       } flatMap { _ =>
```

Comment:
This `flatMap` chain will be much more readable as a for-comprehension as the checks list and indentation have grown so much since we'd first worked on this.